.github/ISSUE_TEMPLATE/issue.yml

```diff
@@ -36,3 +48,14 @@ body:
       description: A short description on how you expect the code to behave.
     validations:
       required: true
+
+  - type: checkboxes
+    id: terms
+    attributes:
+      label: Code of Conduct
+      description: >
+        By submitting this issue, you agree to follow our
+        [Code of Conduct](https://github.com/elixir-lang/elixir/blob/main/CODE_OF_CONDUCT.md#code-of-conduct).
+      options:
+        - label: I agree to follow this project's Code of Conduct
+          required: true
```

Comment:
```suggestion
```

GitHub already prompts and points new users about the code of conduct, so this should be done. Because it is not only about filling issues, it is about any interaction (comments, etc).